core/src/test/java/com/sequenceiq/cloudbreak/core/flow2/externaldatabase/ExternalDatabaseServiceTest.java

```diff
@@ -382,9 +376,8 @@ void provisionDatabaseTestSslWhenNotEntitled() throws JsonProcessingException {
         blueprint.setBlueprintText(BLUEPRINT_TEXT);
         when(cmTemplateProcessorFactory.get(BLUEPRINT_TEXT)).thenReturn(cmTemplateProcessor);
         when(cmTemplateProcessor.getStackVersion()).thenReturn(STACK_VERSION_GOOD);
-        when(entitlementService.databaseWireEncryptionDatahubEnabled(ACCOUNT_ID)).thenReturn(false);
 
-        provisionDatabaseTestSslInternal(blueprint, false);
+        provisionDatabaseTestSslInternal(blueprint, true);
```

Comment:
Instead of setting this, the whole method (`provisionDatabaseTestSslWhenNotEntitled()`) can be removed altogether. The positive case is tested is the method below (`provisionDatabaseTestSslWhenSslEnabled()`).